debugging/coreclr/dockerManager.ts

```diff
@@ -341,18 +341,26 @@ export class DefaultDockerManager implements DockerManager {
             permissions: 'ro'
         };
 
-        let programFilesEnvironmentVariable: string | undefined;
-
-        if (this.osProvider.os === 'Windows') {
-            programFilesEnvironmentVariable = this.processProvider.env[DefaultDockerManager.ProgramFilesEnvironmentVariable];
+        let nugetFallbackFolder: string;
+        if (this.osProvider.isMac) {
+            nugetFallbackFolder = MacNuGetPackageFallbackFolderPath;
```

Comment:
Perhaps `MacNuGetPackageFallbackFolderPath` should just be the .NET installation folder, so that we can build the `nugetFallbackFolder` path in the same way across all the platforms.  Alternatively, just define the constant for Linux (like Mac) and only actually build a path in the case of Windows.

It just feels a little inconsistent that Mac is treated differently than Linux, when both are static paths.

Reply:
Thanks for taking a look. Both ways are possible. I decided not to make ```MacNuGetPackageFallbackFolderPath``` relative because it is exported and used somewhere else. Changing its meaning would've meant having to duplicate the knowledge about where to find the fallback folder relative to the sdk in that location too. It would make things much more uniform though.

I also considered just adding a Linux version of the variable but didn't see a reason for it to be exported or in that scope.

It seemed like things would likely develop away from static paths more towards whatever the dotnet binary does for dynamic sdk detection on the different platforms so the patch goes into that direction.

I think the smallest change to just achieve the fix would be to just use an explicit string for linux and the existing behavior for the rest. If that's preferrable I can adjust the PR to that.

Reply:
Yeah, I see the Mac path used in a pre-requisite check as well.  Ok, I see a couple of options of varying complexity:

1. As you suggest, just make the complete Linux path a constant, in the same way as the Mac path, defined next to the Mac path (whether exported or not).  This has the advantage of making the differences between those platforms more obvious, being able to see the paths next to each other.  Obviously the simplest change.

1. Being a .NET-specific property, one could argue for a new property on the `DotNetClient` interface (e.g. `nuGetPackageFallbackFolderPath: string`) which builds the path based on the local platform OS.  That logic can then be easily shared between the `DockerManager` and the pre-requisite checks.  A bigger change, but feels like the "better" one to me.

It'd be great to do one of these; I'll let you decide which, though.

Reply:
Went for option 1 .